.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/sicl"]
+	path = vendor/sicl
+	url = git@github.com:robert-strandh/SICL.git
```

Comment:
This particular URL gives me the following error when running `git submodule update`:

````
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Clone of 'git@github.com:robert-strandh/SICL.git' into submodule path 'vendor/sicl' failed
````

It fetches correctly if I change it to fetch over https instead: `https://github.com/robert-strandh/SICL.git`